config/core/configmaps/autoscaler.yaml

```diff
@@ -137,3 +137,12 @@ data:
     # that should be used if none is specified. If omitted, the Knative
     # Horizontal Pod Autoscaler (KPA) is used by default.
     pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+
+    # scale-to-zero-on-deploy creates new revisions with zero initial
+    # replicas.  With this setting enabled the user container is not
+    # activated, nor are readinessProbes evaluated before the new
+    # Revision is declared Ready.  This setting should only be enabled
+    # with extreme caution as it may cause traffic to shift to unverified
+    # Revisions.  This behavior is overridden when the minScale
+    # annotation is specified.
+    scale-to-zero-on-deploy: "false"
```

Comment:
we can bikeshed a bit here, since we're not really "scaling to 0 on deploy" but rather"skip-revision-startup-validation"-ing.
As in `scale-to-zero-on-deploy` presumes you scaled to `>0` first and now you scale to 0.